assets/js/modules/globalNav.js

```diff
@@ -24,6 +24,7 @@ define([
         "showPreviews": false,
         "sortType": "sortByDate",
         "sortDirection":"forward",
+        "delimeter": " ",
```

Comment:
I think it should be comma by default. Could you please also add some info about this in generated nav documentation https://github.com/sourcejs/Source/blob/master/docs/data-nav/readme.md?